fractal-puppy/Engine
# Feature/background sounds

Added background sounds for both levels. I also updated both scenes.
To test:
- see that the level 1 now has thunder sounds. (source: https://www.youtube.com/watch?v=ZvSwho5CRTo)
- see that level 2 has background sewer and rats sounds. (source: https://www.youtube.com/watch?v=pv7sc85UZ-4)

- see also that when starting the bossfight the sewer sounds stop.
